projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -642,4 +642,10 @@ export interface OccEndpoints {
    * @member {string}
    */
   orderApprovalDecision?: string | OccEndpoint;
+  /**
+   * Endpoint for gettting base sites
```

Comment:
Typo. I'd start using `Spelling Checker for Visual Studio Code` a while back which is really helping me to find spelling mistakes.